Classes/WPLegacyEditorViewController.m

```diff
@@ -7,27 +7,34 @@
 CGFloat const WPLegacyEPVCToolbarHeight = 44.0;
 
 @interface WPLegacyWrapperViewForInputView: UIView
-    @property (nonatomic, strong) UIToolbar *toolbar;
+    @property (nonatomic, strong) WPLegacyEditorFormatToolbar *toolbar;
 @end
 
 @implementation WPLegacyWrapperViewForInputView
 
-- (instancetype)initWithToolbar:(UIToolbar *)toolbar {
-    self = [super initWithFrame:CGRectMake(0, 0, self.frame.size.width, toolbar.frame.size.height)];
+- (instancetype)initWithToolbar:(WPLegacyEditorFormatToolbar *)toolbar {
+    self = [super initWithFrame:CGRectMake(0, 0, self.frame.size.width, WPLegacyEPVCToolbarHeight)];
     if (self) {
         _toolbar = toolbar;
         self.autoresizingMask = UIViewAutoresizingFlexibleHeight;
-        self.backgroundColor = [[WPLegacyEditorFormatToolbar appearance] backgroundColor];
+        self.backgroundColor = toolbar.backgroundColor ? toolbar.backgroundColor : [[WPLegacyEditorFormatToolbar appearance] backgroundColor];
```

Comment:
FYI you can shortcut this to `toolbar.backgroundColor ?: [[WPLegacyEditorFormatToolbar appearance] backgroundColor];`